packages/editor/src/components/inserter/menu.js

```diff
@@ -281,13 +278,20 @@ export class InserterMenu extends Component {
 
 					{ !! reusableItems.length && (
 						<PanelBody
+							className="editor-inserter__reusable-blocks-panel"
 							title={ __( 'Reusable' ) }
 							opened={ isPanelOpen( 'reusable' ) }
 							onToggle={ this.onTogglePanel( 'reusable' ) }
 							icon="controls-repeat"
 							ref={ this.bindPanel( 'reusable' ) }
 						>
 							<BlockTypesList items={ reusableItems } onSelect={ onSelect } onHover={ this.onHover } />
+							<IconButton
+								className="editor-inserter__manage-reusable-blocks"
+								icon="admin-generic"
+								label={ __( 'Manage All Reusable Blocks' ) }
+								href="edit.php?post_type=wp_block"
```

Comment:
For a later PR. We use links to target WP Admin pages (here, revisions, I'm probably missing other links too). The thing is. These links only work in WP context. We need to find a way to add these links in an "augmented" manner. Keeping the packages WP links free.

Reply:
Yes, let's make sure we make an issue for this.

Reply:
#9637